recipes/khiopsdriver-gcs/recipe.yaml

```diff
@@ -0,0 +1,50 @@
+context:
+  version: "0.0.12"
+
+package:
+  name: khiops-driver-gcs
+  version: ${{ version }}
+
+source:
+  url: https://github.com/KhiopsML/khiopsdriver-gcs/archive/refs/tags/${{ version }}.tar.gz
+  sha256: 456daa8f441bf2e461b73949c87307dd7f20ed1e62c4a334a6d4dbcf094cc713
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib("c") }}
+    - cmake
+    - ninja
+  host:
+    - fmt
+    - spdlog
+    - zlib
+    - libgoogle-cloud-storage-devel
+    - libboost-devel
+    - libboost-headers
+    - libcrc32c
+    - if: win
+      then: libcurl
+
+tests:
+  - script:
+    - if: linux
+      then: test -f $PREFIX/lib/libkhiopsdriver_file_gcs.so
+    - if: osx
+      then: test -f $PREFIX/lib/libkhiopsdriver_file_gcs.dylib
+    - if: win
+      then: if not exist %PREFIX%\\Library\\bin\\libkhiopsdriver_file_gcs.dll exit 1
```

Comment:
```suggestion
    - if: unix
      then: test -f $PREFIX/lib/libkhiopsdriver_file_gcs${SHLIB_EXT}
      else: 
        - if not exist %LIBRARY_PREFIX%\bin\libkhiopsdriver_file_gcs.dll exit 1 
        - if not exist %LIBRARY_PREFIX%\lib\libkhiopsdriver_file_gcs.lib exit 1
```

Reply:
Actually the generated dll is not used by anything else than Khiops, which is why we do not generate the .lib